backend/config/curation-api.yml

```diff
@@ -356,7 +356,46 @@ paths:
           $ref: "#/components/responses/401"
         "404":
           $ref: "#/components/responses/404"
-
+  /v1/collections/{collection_id}/datasets/s3-upload-credentials/submission_key:
+    put:
```

Comment:
I don't think this should be a `PUT`. I think it should be a `GET`.

This call does not modify any resource. It requests information about a resource.

Reply:
Since there is a request body it needed to be a `PUT`, but a `GET` does make more sense. The parameters will need to query parameters in this case. Not a big problem, but the tag or dataset_id will need to be made URL friendly.